packages/storefront-lwc/src/modules/commerce/basket/basket.html

```diff
@@ -21,50 +18,48 @@
 
         <template if:false={loading}>
             <template if:false={hasProducts}>
-                <div class="container">
-                    <div class="row  pt-5">
-                        <div class="col">
-                            <p class="text-center pt-2"> 0 Items </p>
-                        </div>
+                <div class="row pt-5">
+                    <div class="col">
+                        <p class="text-center pt-2"> 0 Items </p>
                     </div>
-                    <div class="row pt-5">
-                        <div class="col text-center">
-                            <h3> Your Shopping Cart is Empty </h3>
-                        </div>
+                </div>
+                <div class="row pt-5">
+                    <div class="col text-center">
+                        <h3> Your Shopping Cart is Empty </h3>
```

Comment:
If we do not have an h1 and an h2 on the page already having an h3 here is an accessibility issue